### ISSUE TITLE
image to Signature

### ISSUE THREAD

**Issue by thediex** [thomasjbradley/signature-to-image#10]
It would be possible to have the image of the signature you can give the json in format {"lx" :, "ly" :, "mx" :, "my":}

**Comment by thomasjbradley** [comment]
Yes, this library will take the JSON in that format and convert it to an image.

**Comment by thediex** [comment]
Thank you very much for your prompt response. For my part I have already been able to convert it into an image, but what I need is to convert an image that I have, in the format to store it in my database.

**Comment by thomasjbradley** [comment]
The reverse conversion? Take an image and turn it back into the JSON data? That’s not possible to do. I generally keep the JSON data and store that in the database.
